js/modules/loki_app_dot_net_api.js

```diff
@@ -62,6 +62,33 @@ class LokiAppDotNetAPI extends EventEmitter {
     return thisServer;
   }
 
+  static async getServerTime() {
+    const url = `${window.getDefaultFileServer()}/loki/v1/time`;
+    let timestamp;
```

Comment:
```suggestion
    let timestamp = NaN;
```
We can just initialise the timestamp to NaN here incase the node fetch request fails